### PR TITLE
Add PIP-195 to 3.0 release note

### PR DESCRIPTION
**Pull request by coderzc**
### Documentation

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

File: release-notes/versioned/pulsar-3.0.0.md [hunk 1]
```diff
@@ -68,49 +69,59 @@ sidebar_label: Apache Pulsar 3.0.0
...
+  - Add metrics for bucket delayed message tracker by @coderzc in [#19716](https://github.com/apache/pulsar/pull/19716)
+  - Don't clean up BucketDelayedDeliveryTracker when all consumer disconnect by @coderzc in [#19801](https://github.com/apache/pulsar/pull/19801)
+  - Add topicName and cursorName for ledger metadata of bucket snapshot by @coderzc in [#19802](https://github.com/apache/pulsar/pull/19802)
+  - Make bucket merge operation asynchronous by @coderzc in [#19873](https://github.com/apache/pulsar/pull/19873)
+  - Clear delayed message when unsubscribe &  Make clear operation asynchronous by @coderzc in [#19901](https://github.com/apache/pulsar/pull/19901)
+  - Merge multiple buckets at once by @coderzc in [#19927](https://github.com/apache/pulsar/pull/19927)
+  - Ensure previous delayed index be removed from snapshotSegmentLastIndexTable & Make load operate asynchronous by @coderzc in [#20086](https://github.com/apache/pulsar/pull/20086)
+  - Fix avoid future of clear delayed message can't complete by @coderzc in [#20075](https://github.com/apache/pulsar/pull/20075)
```
Comment (Anonymitaet) [inline release-notes/versioned/pulsar-3.0.0.md:92]:
```suggestion
  - Fix the issue of delayed messages can't be completed by @coderzc in [#20075](https://github.com/apache/pulsar/pull/20075)
```
Do you mean this?

**Review by Anonymitaet — COMMENTED** [review]
Thanks for your contribution! Have you previewed your changes and ensured everything goes as expected? 

If not, please [preview your changes locally](https://pulsar.apache.org/contribute/document-preview/#preview-changes) and attach the screenshots in this PR. In this way, you can get your PR merged more quickly. Thanks!

**Comment by coderzc** [comment]
> Thanks for your contribution! Have you previewed your changes and ensured everything goes as expected?
> 
> If not, please [preview your changes locally](https://pulsar.apache.org/contribute/document-preview/#preview-changes) and attach the screenshots in this PR. In this way, you can get your PR merged more quickly. Thanks!

@Anonymitaet I already previewed these changes and the result is expected.

image

File: release-notes/versioned/pulsar-3.0.0.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ sidebar_label: Apache Pulsar 3.0.0
   - Clear delayed message when unsubscribe &  Make clear operation asynchronous by @coderzc in [#19901](https://github.com/apache/pulsar/pull/19901)
   - Merge multiple buckets at once by @coderzc in [#19927](https://github.com/apache/pulsar/pull/19927)
   - Ensure previous delayed index be removed from snapshotSegmentLastIndexTable & Make load operate asynchronous by @coderzc in [#20086](https://github.com/apache/pulsar/pull/20086)
-  - Fix avoid future of clear delayed message can't complete by @coderzc in [#20075](https://github.com/apache/pulsar/pull/20075)
+  - Fix the issue which the `future` of clean up delayed messages can't be completed by @coderzc in [#20075](https://github.com/apache/pulsar/pull/20075)
```
Comment (Anonymitaet) [inline release-notes/versioned/pulsar-3.0.0.md:92]:
```suggestion
  - Fix the issue of the `future` of clean-up delayed messages can't be completed by @coderzc in [#20075](https://github.com/apache/pulsar/pull/20075)
```

**Review by RobertIndie — APPROVED** [review]

**Pull request merged**